core/appStatusPolling/appStatusPooling_test.go

```diff
@@ -0,0 +1,65 @@
+package appStatusPolling_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core/appStatusPolling"
+	"github.com/ElrondNetwork/elrond-go/core/mock"
+	"github.com/ElrondNetwork/elrond-go/statusHandler"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewAppStatusPooling_NilAppStatusHandlerShouldErr(t *testing.T) {
```

Comment:
these tests can be made to run in parallel.

Reply:
done